config/blobs.yml

```diff
@@ -78,6 +78,10 @@ nginx/nginx-1.25.2.tar.gz:
   size: 1214903
```

Comment:
The previous version needs to be removed, otherwise the [unused_blobs](https://github.com/cloudfoundry/capi-ci/blob/main/ci/bosh/unused_blobs.rb) script that runs as part of the [create-release-candidate](https://concourse.app-runtime-interfaces.ci.cloudfoundry.org/teams/capi-team/pipelines/capi/jobs/create-release-candidate) job will fail.